src/main/java/com/prgrms/board/dto/comment/CommentSaveRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.prgrms.board.dto.comment;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+public class CommentSaveRequest {
+    @NotNull(message = "유저 Id는 필수 요소입니다.")
+    @Min(value = 1, message = "유저 Id는 1 이상이어야 합니다.")
```

Comment:
```suggestion
    @Positive(message = "유저 Id는 1 이상이어야 합니다.")
```
양수로 제한한다면 이런 방법도 있습니다~